ts/nni_manager/core/nniDataStore.ts

```diff
@@ -59,7 +59,7 @@ class NNIDataStore implements DataStore {
         try {
             await this.db.storeExperimentProfile(experimentProfile);
         } catch (err) {
-            throw NNIError.FromError(err, 'Datastore error: ');
+            throw NNIError.FromError(err as any, 'Datastore error: ');
```

Comment:
why we need add these `as any`

Reply:
Because tsc has been updated and is complaining.
The code is somewhat wrong. The function only accepts Error and string, but there is no such guarantee for err's type. Don't want to fix it though.